cli/src/main.rs

```diff
@@ -244,10 +249,43 @@ fn compile_once(world: &mut SystemWorld, command: &CompileSettings) -> StrResult
     world.main = world.resolve(&command.input).map_err(|err| err.to_string())?;
 
     match typst::compile(world) {
-        // Export the PDF.
+        // Export the PDF / PNG.
         Ok(document) => {
-            let buffer = typst::export::pdf(&document);
-            fs::write(&command.output, buffer).map_err(|_| "failed to write PDF file")?;
+            match command.output.extension() {
+                Some(ext) if ext.eq_ignore_ascii_case("png") => {
+                    let pixel_per_pt = command.ppi.unwrap_or(2.0);
+                    let pixmaps: Vec<_> = document
+                        .pages
+                        .iter()
+                        .map(|frame| {
+                            typst::export::render(frame, pixel_per_pt, Color::WHITE)
+                        })
+                        .collect();
+
+                    if pixmaps.len() == 1 {
+                        pixmaps[0]
+                            .save_png(command.output.clone())
+                            .map_err(|_| "failed to write PNG file")?;
+                    } else {
```

Comment:
Ah right, I didn't think about that. In my head this made more sense since it would be weird for the image export to change the name of the exported file if there was more than one page but maybe we make pagination an option?